src/services/account/useUpgradePlan.ts

```diff
@@ -24,12 +25,12 @@ export function useUpgradePlan({ provider, owner }: useUpgradePlanParams) {
   }
 
   return useMutation({
-    mutationFn: (formData: { seats: number; newPlan: string }) => {
+    mutationFn: (formData: { seats: number; newPlan: Plan }) => {
       const path = getPathAccountDetails({ provider, owner })
       const body = {
         plan: {
           quantity: formData?.seats,
-          value: formData?.newPlan,
+          value: formData?.newPlan.value,
```

Comment:
You can see that even though we're passing in a Plan, we haven't modified the mutation object that the backend receives at the end of the day